packages/server/src/index.ts

```diff
@@ -35,12 +42,12 @@ export function verifyPrivacy(properties: Property[], sign: Signature, publicKey
   return verifyAccordingToKeyType(merkleRoot, sign.value, publicKey.key, publicKey.type);
 }
 
-// TODO: @ts-ignore should be removed, when below function is implemented.
-// @ts-ignore
 export function verifyPackage(
   properties: Property[],
   sign: Signature,
   publicKey: PublicKey
 ): boolean {
-  // TODO: Not implemented yet.
+  const encodings = properties.map(property => utf8ToBase64(property.key)+":"+utf8ToBase64(property.value)),
+    hashValue = hash(encodings.join(','));
+  return verifyAccordingToKeyType(hashValue, sign.value, publicKey.key, publicKey.type);
```

Comment:
After simply processing base64 encoding, the signature is verified through the hashed value.